src/nsrt_learning/sampler_learning.py

```diff
@@ -312,12 +309,6 @@ def sampler(self, state: State, goal: Set[GroundAtom],
                self._classifier.classify(np.r_[x, params]):
                 break
             num_rejections += 1
-        else:
-            # Edge case: we exceeded the number of sampling tries
-            # and we might be left with a params that is not in
-            # bounds. If so, fall back to sampling from the space.
-            if not self._param_option.params_space.contains(params):
-                params = self._param_option.params_space.sample()
```

Comment:
note: we're changing the behavior from random sample to clip here. shouldn't matter hopefully